prompts/sources.go

```diff
@@ -120,14 +126,56 @@ func sourceBaseForm(quickstart *Quickstart) (*QuickstartState, error) {
 		fileLocation = *quickstart.Defaults.SchemaPath
 	}
 
-	if _, err := charm_internal.NewForm(huh.NewForm(
-		getBaseSourcePrompts(quickstart.WorkflowFile, &sourceName, &fileLocation, &authHeader, &authSecret)...),
-		"Let's setup a new source for your workflow.",
-		"A source is a compiled set of OpenAPI specs and overlays that are used as the input for a SDK generation.").
-		ExecuteForm(); err != nil {
+	useSampleSpec := false
+	_, err := charm_internal.NewForm(huh.NewForm(
```

Comment:
Let's maybe have a title here. Something like an OpenAPI spec is used to generate your SDK target. You can use either a local spec or a remote one.

Something like that

Reply:
Basically what I want to avoid is a user not having their spec available locally and thinking that means they need to use the sample spec.

Reply:
I understand, that makes sense, I've updated to add a description do you think that's clear enough?